storage/pipeline/input.go

```diff
@@ -341,14 +350,25 @@ func (m *Sealing) SectorAddPieceToAny(ctx context.Context, size abi.UnpaddedPiec
 	return api.SectorOffset{Sector: res.sn, Offset: res.offset.Padded()}, res.err
 }
 
+func (m *Sealing) getClaimTerms(ctx context.Context, deal api.PieceDealInfo) (pieceClaimBounds, error) {
+	// TODO: TODO! TODO! get the real claim bounds here
```

Comment:
If you want to defer the work of doing this "the right way" then you can compute these just like f05: https://github.com/filecoin-project/builtin-actors/blob/master/actors/market/src/lib.rs#L1143
https://github.com/filecoin-project/builtin-actors/blob/master/runtime/src/runtime/policy.rs#L408
Its not really any more brittle than "the right way" since the next change to the market <> miner interface is probably going to break both.

Like you probably know already "the right way" is 1. go to market to read map of pending deals to allocation ids 2. go to verifreg and read the allocation table with provider / allocation id

Reply:
I've already implemented the API to do this "the right way" so I think it's better to do that.